components/Card/index.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledCard = styled.div`
+  border: 1px solid #ddd;
+  border-radius: 10px;
+  padding: 20px;
+  margin: 20px;
+  cursor: pointer;
+`;
+
+const ProjectTitle = styled.h3`
+  margin: 0;
+`;
+
+const OtherValuesContainer = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
```

Comment:
```suggestion
```
I don't think these two lines are necessary if you're giving your `OtherValues` component a `height` of `fit-content`.
You can also work more with `gap` here instead of adding a gap via margin in your `OtherValues` component.